### PR TITLE
removed trailing "|" in language

### PR DESCRIPTION
**Pull request by silv3rr**

### CODE REVIEW

File: scripts/psxc-imdb/main/psxc-imdb.sh [hunk 1]
```diff
@@ -811,9 +811,9 @@ if [ ! -z "$RUNCONTINOUS" ] || [ -z "$RECVDARGS" ]; then
         MYOWNFORMAT1="$(echo "${MYOWNFORMAT1}" | sed "s^$MYOWNSTRING^$MYOWNEMPTY^g")"
        fi
       done
-      echo "$DATE $TRIGGER \"$IMDBLKL\" \"${MYOWNFORMAT1}\" \"$IMDBDST\"" | tr '[=$=]' '�' | sed "s|�|USD|g" >> $GLLOG
+      echo "$DATE $TRIGGER \"$IMDBLKL\" \"${MYOWNFORMAT1}\" \"$IMDBDST\"" | tr '[=$=]' '¤' | sed "s|¤|USD|g" >> $GLLOG
      else
-      echo "$DATE $TRIGGER \"$IMDBLKL\" \"$IMDBDIR\" \"$IMDBURL\" \"$TITLE\" \"$GENRECLEAN\" \"$RATINGCLEAN\" \"$COUNTRYCLEAN\" \"$LANGUAGECLEAN\" \"$CERTCLEAN\" \"$RUNTIMECLEAN\" \"$DIRECTORCLEAN\" \"$BUSINESSSHORT\" \"$PREMIERE\" \"$LIMITED\" \"$RATINGVOTES\" \"$RATINGSCORE\" \"$TITLENAME\" \"$TITLEYEAR\" \"$BUSINESSSCREENS\" \"$ISLIMITED\" \"$CASTLEADNAME\" \"$CASTLEADCHAR\" \"$TAGLINECLEAN\" \"$PLOTCLEAN\" \"$RATINGBAR\" \"$CASTCLEAN\" \"$COMMENTSHORTCLEAN\" \"$IMDBDST\"" | tr '[=$=]' '�' | sed "s|�|USD|g" >> $GLLOG
```
Comment (silv3rr) [inline scripts/psxc-imdb/main/psxc-imdb.sh]: ignore

File: scripts/psxc-imdb/main/psxc-imdb.sh [hunk 2]
```diff
@@ -771,7 +771,7 @@ if [ ! -z "$RUNCONTINOUS" ] || [ -z "$RECVDARGS" ]; then
      echo "$DATE $TRIGGER \"$IMDBLKL\" \"$RUNTIME\" \"$IMDBDST\"" >> $GLLOG
     fi
     if [ ! -z "$BUSINESSSHORT" ] && [ -z "$BOTONELINE" ]; then
-     echo "$DATE $TRIGGER \"$IMDBLKL\" \"Opening Stats: $BUSINESSSHORT\" \"$IMDBDST\"" | tr '[=$=]' '�' | sed "s|�|USD|g" >> $GLLOG
```
Comment (silv3rr) [inline scripts/psxc-imdb/main/psxc-imdb.sh]: ignore

File: scripts/psxc-imdb/main/psxc-imdb.sh [hunk 3]
```diff
@@ -811,9 +811,9 @@ if [ ! -z "$RUNCONTINOUS" ] || [ -z "$RECVDARGS" ]; then
         MYOWNFORMAT1="$(echo "${MYOWNFORMAT1}" | sed "s^$MYOWNSTRING^$MYOWNEMPTY^g")"
        fi
       done
-      echo "$DATE $TRIGGER \"$IMDBLKL\" \"${MYOWNFORMAT1}\" \"$IMDBDST\"" | tr '[=$=]' '�' | sed "s|�|USD|g" >> $GLLOG
```
Comment (silv3rr) [inline scripts/psxc-imdb/main/psxc-imdb.sh]: ignore

File: scripts/psxc-imdb/main/psxc-imdb.sh (hunk 1 above)
Comment (silv3rr) [inline scripts/psxc-imdb/main/psxc-imdb.sh]: ignore

File: scripts/psxc-imdb/main/psxc-imdb.sh (hunk 3 above)
Comment (silv3rr) [inline scripts/psxc-imdb/main/psxc-imdb.sh]: ignore

**Comment by Skeddie** [comment]
Update your patch to remove those changes then. Also integrate the 3 'sed's into one: "sed 's/[ /|]*$//'".
Use "git rebase -i HEAD~2" to squash/fixup the second commit into the previous and then a "git push -f".

**Comment by Skeddie** [comment]
No reaction for over 2 months, closing and doing changes manually.

**Pull request closed without merging**